src/fsipy/automatedPostprocessing/postprocessing_h5py/spectrograms.py

```diff
@@ -0,0 +1,778 @@
+
+    parser.add_argument("--folder", type=Path, required=True, default=None,
+                        help="Path to simulation results")
+    parser.add_argument('--mesh-path', type=Path, default=None,
+                        help="Path to the mesh file (default: <folder_path>/Mesh/mesh.h5)")
+    parser.add_argument('--save-deg', type=int, default=2,
+                        help="Specify the save_deg used during the simulation, i.e., whether the intermediate P2 nodes "
+                             "were saved. Entering save_deg=1 when the simulation was run with save_deg=2 will result "
+                             "in using only the corner nodes in postprocessing.")
+    parser.add_argument('--stride', type=int, default=1,
+                        help="Desired frequency of output data (i.e. to output every second step, use stride=2)")
+    parser.add_argument('--start-time', type=float, default=0.0,
+                        help="Start time of the simulation (in seconds).")
+    parser.add_argument('--end-time', type=float, default=0.05,
+                        help="End time of the simulation (in seconds).")
+    parser.add_argument('--lowcut', type=float, default=25,
+                        help="Cutoff frequency (Hz) for the high-pass filter.")
+    parser.add_argument('--ylim', type=float, default=800,
+                        help="Set the y-limit of the spectrogram graph.")
+    parser.add_argument('--sampling-region', type=str, default="sphere",
+                        help="Specify the sampling region. Choose 'sphere' to sample within a sphere or 'domain' to "
+                             "sample within a specified domain.")
+    parser.add_argument('--fluid-sampling-domain-id', type=int, default=1,
+                        help="Domain ID for the fluid region to be sampled. Input a labelled mesh with this ID.")
+    parser.add_argument('--solid-sampling-domain-id', type=int, default=2,
+                        help="Domain ID for the solid region to be sampled. Input a labelled mesh with this ID.")
+    parser.add_argument('--r-sphere', type=float, default=1000000,
+                        help="Radius of the sphere used to include points for spectrogram.")
+    parser.add_argument('--x-sphere', type=float, default=0.0,
+                        help="X-coordinate of the center of the sphere used to include points for spectrogram (in "
+                             "meters).")
+    parser.add_argument('--y-sphere', type=float, default=0.0,
+                        help="Y-coordinate of the center of the sphere used to include points for spectrogram (in "
+                             "meters).")
+    parser.add_argument('--z-sphere', type=float, default=0.0,
+                        help="Z-coordinate of the center of the sphere used to include points for spectrogram (in "
+                             "meters).")
+    parser.add_argument('--dvp', type=str, default="v",
```

Comment:
I would like `dvp` to be more general since it can technically be `wss` or other variables that are not included `dvp`. Something like `-voi, --variable-of-interest` is more intuitive to me.

Reply:
@keiyamamo - How about `--quantity`:
``` python
    parser.add_argument('-q', '--quantity', type=str, default="v",
                        help="Quantity to postprocess. Choose 'v' for velocity, 'd' for displacement, 'p' for "
                             "pressure, or 'wss' for wall shear stress.")
```
Is that okay with you? Or do you prefer your suggestion?

Reply:
Quantity sounds good to me! Please go ahead with that.